### PR TITLE
Execute oVirt deployment playbook as a part of tests

### PR DESCRIPTION
**Pull request by lukas-bednar**
Fix issue #56

### CODE REVIEW

File: roles/ovirt-engine-setup/README.md [hunk 1]
```diff
@@ -44,6 +44,9 @@ ovirt_engine_configure_iso_domain: Whether to confiure ISO domain on engine (def
 ovirt_engine_iso_domain_path: Create local ISO domain on engine machine (default: /var/lib/exports/iso)
 ovirt_engine_iso_domain_name: Name of ISO domain (default: 'ISO_DOMAIN')
 ovirt_engine_iso_domain_acl: ACL permissions for ISO domain mount point (default: '0.0.0.0/0.0.0.0(rw)')
+
+# Configure firewall
+ovirt_engine_firewall_manager: In case you want to configure firewall, select the firewall manager ('firewalld', 'iptables', default=null)
```
Comment (StLuke) [inline roles/ovirt-engine-setup/README.md]: default should be the same as for engine-setup (firewalld)

File: roles/ovirt-engine-setup/defaults/main.yml [hunk 2]
```diff
@@ -20,3 +20,5 @@ ovirt_engine_configure_iso_domain: false
 ovirt_engine_iso_domain_path: '/var/lib/exports/iso'
 ovirt_engine_iso_domain_name: 'ISO_DOMAIN'
 ovirt_engine_iso_domain_acl: '0.0.0.0/0.0.0.0(rw)'
+
+ovirt_engine_firewall_manager: null
```
Comment (StLuke) [inline roles/ovirt-engine-setup/defaults/main.yml]: s/null/firewalld/
as this is deafult for engine-setup, if you dont want override

File: roles/ovirt-engine-setup/README.md (hunk 1 above)
Comment (lukas-bednar) [inline roles/ovirt-engine-setup/README.md]: Done

File: roles/ovirt-engine-setup/defaults/main.yml (hunk 2 above)
Comment (lukas-bednar) [inline roles/ovirt-engine-setup/defaults/main.yml]: Done

File: roles/ovirt-engine-setup/README.md [hunk 3]
```diff
@@ -44,6 +44,11 @@ ovirt_engine_configure_iso_domain: Whether to confiure ISO domain on engine (def
 ovirt_engine_iso_domain_path: Create local ISO domain on engine machine (default: /var/lib/exports/iso)
 ovirt_engine_iso_domain_name: Name of ISO domain (default: 'ISO_DOMAIN')
 ovirt_engine_iso_domain_acl: ACL permissions for ISO domain mount point (default: '0.0.0.0/0.0.0.0(rw)')
+
+# Configure firewall
+ovirt_engine_firewall_manager: |
+  In case you want to configure firewall, select the firewall manager
+  'firewalld', 'iptables' (default='firewalld'), or put null otherwise.
```
Comment (petr-balogh) [inline roles/ovirt-engine-setup/README.md]: I think you cannot pass null as a value of variable, only none.
null you can use in defautl variables in defaults/main.yml

Comment (lukas-bednar) [inline roles/ovirt-engine-setup/README.md]: null is None in yaml world ...
```
In [1]: import yaml

In [3]: from StringIO import StringIO

In [4]: s = StringIO()

In [5]: s.write("my_var: null\n")

In [7]: s.seek(0)

In [8]: yaml.load(s)
Out[8]: {'my_var': None}
```

Comment (petr-balogh) [inline roles/ovirt-engine-setup/README.md]: Ok, my fault, I thought that I met some issue when defining null or none in variable file, but don't remember and cannot reproduce any now :+1:

**Pull request merged**